Sources/Sentry/SentryScope.m

```diff
@@ -422,13 +422,13 @@ - (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
             subarrayWithRange:NSMakeRange(0, MIN(maxBreadcrumbs, [breadcrumbs count]))];
     }
 
+    NSMutableDictionary *newContext;
```

Comment:
`l`: Can we maybe put the whole context code closer together? We could move these few lines of code down to L:466.